Users/Docker_Image.rst

```diff
@@ -45,3 +45,23 @@ for the ``gitlab-ci-multi-runner`` runner:
 Please be aware that the most generic ``dns`` setting listed in the
 ``gitlab-ci-multi-runner`` documentation has been recently added and at
 the time of this writing is not available in official builds.
+
+coala on Travis CI
+------------------
+
+You can use the ``coala/base`` docker image to perform static code analysis
+on your code with a ``.travis.yml``, like this:
+
+::
+
+    language: generic
+    sudo: required
+    services:
+      - docker
+    script:
+      - docker run -v $(pwd):/app -w /app coala/base coala-ci
```

Comment:
we may want to note that this same this very command can be used for any CI where docker is present so this isn't limited to travis